Always include <wx/timer.h> since it's now in the base lib

Signed-Off-By: Anders Larsen <al@alarsen.net>
---
 include/wx/wx.h |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

Index: b/include/wx/wx.h
===================================================================
--- a/include/wx/wx.h
+++ b/include/wx/wx.h
@@ -29,6 +29,7 @@
 #include "wx/memory.h"
 #include "wx/math.h"
 #include "wx/stopwatch.h"
+#include "wx/timer.h"
 #include "wx/module.h"
 #include "wx/wxcrt.h"
 #include "wx/wxcrtvararg.h"
@@ -61,7 +62,6 @@
 #include "wx/icon.h"
 #include "wx/cursor.h"
 #include "wx/dialog.h"
-#include "wx/timer.h"
 #include "wx/settings.h"
 #include "wx/msgdlg.h"
 #include "wx/cmndata.h"

 	  	 
